CHANGELOG.md

```diff
@@ -4,6 +4,9 @@ Please, check out guidelines: https://keepachangelog.com/en/1.0.0/
 
 ## Next
 
+### Removed
```

Comment:
Is it worth marking this as "breaking" ? 

For anyone that did a `tuist init` in prior versions and upgrades to this version, `tuist generate` may fail as it won't recognise `.generateManifest` option

Reply:
> Is it worth marking this as "breaking" ?

Good point. We should definitively mark this one as breaking.